tests/integration/user_test.go

```diff
@@ -276,3 +278,41 @@ func TestListStopWatches(t *testing.T) {
 		assert.Greater(t, apiWatches[0].Seconds, int64(0))
 	}
 }
+
+func TestGetOpenStreetMapLink(t *testing.T) {
```

Comment:
This test is a no-op for Golang template.

Golang template itself do the "context escaping" for URL and query parameters correctly. All template code is based on this fact.

It shouldn't test it again here.

Reply:
Do you suggest that the test should be removed?

Reply:
Yes, it doesn't make sense to test a Golang template's builtin mechanism. All URLs in templates are constructed this way.